src/Microsoft.DotNet.AsmDiff/DiffCSharpWriter.cs

```diff
@@ -193,25 +193,38 @@ public override void Visit(IEnumerable<MemberMapping> members)
 
         public override void Visit(MemberMapping member)
         {
-            IDisposable style = null;
-
-            if (this.HighlightBaseMembers)
+            if (!IsPropertyOrEventAccessor(member.Representative))
             {
-                if (member.Representative.IsInterfaceImplementation())
-                    style = _syntaxWriter.StartStyle(SyntaxStyle.InterfaceMember);
-                else if (member.Representative.IsOverride())
-                    style = _syntaxWriter.StartStyle(SyntaxStyle.InheritedMember);
-            }
+                IDisposable style = null;
 
-            WriteHeader(member);
-            if (style != null)
-                style.Dispose();
+                if (this.HighlightBaseMembers)
+                {
+                    if (member.Representative.IsInterfaceImplementation())
+                        style = _syntaxWriter.StartStyle(SyntaxStyle.InterfaceMember);
+                    else if (member.Representative.IsOverride())
+                        style = _syntaxWriter.StartStyle(SyntaxStyle.InheritedMember);
+                }
+
+                WriteHeader(member);
+                if (style != null)
+                    style.Dispose();
+
+                _syntaxWriter.WriteLine();
+                WriteComments(member);
+            }
 
-            _syntaxWriter.WriteLine();
-            WriteComments(member);
             base.Visit(member);
         }
 
+        private static bool IsPropertyOrEventAccessor(ITypeDefinitionMember representative)
+        {
+            var methodDefinition = representative as IMethodDefinition;
+            if (methodDefinition == null)
+                return false;
+
+            return methodDefinition.IsPropertyOrEventAccessor();
```

Comment:
A slightly simpler pattern:
```suggestion
            return (representative is IMethodDefinition methodDefinition) && methodDefinition.IsPropertyOrEventAccessor();
```